devel/external/AEGIS/AEGIS_25.2_training_sample_readme.txt

```diff
@@ -0,0 +1,109 @@
+This README file describes the AEGIS F814W<25.2 training sample, which is a
+randomly ordered sample of galaxies in the AEGIS field with AB magnitude<25.2, in F606W(V) and F814W(I) bands.
+
+NOTE: The columns in the catalog are those expected by galsim.RealGalaxyCatalog.
+However, some parts of that analysis, i.e parametric fits, were not performed.
+Hence those values were manually set and do not have physical meaning.
+
+(1) The catalogs themselves are in AEGIS_galaxy_catalog_[VI]_25.2.fits (where
+[VI] indicates either V or I).  These files can be used to initialize a
```

Comment:
Need to comment on the correspondence between the V and I catalogs: was dual image mode used for measurements (yes), do the entries in the catalogs correspond to each other (I imagine not?), etc.?  For what it's worth, I was expecting a single catalog with information about both bands, where you'd have the same IDENT appearing twice but with different values of 'BAND' and locations of the images.  What is the motivation for separating the bands into different catalogs?  

Note that using such a multi-band catalog that I had envisioned would require some updates to how COSMOSCatalog and RealGalaxyCatalog are used to make images, but it should be possible for them to be backwards compatible.